docs/deployment/sso.md

```diff
@@ -80,59 +80,66 @@ The following references are used:
-    3. Click **Save**.
-4. Under the **Attributes & Claims** form, click **Edit** > **Add new claim**.
-    1. Enter `email` in the **Name** field.
-    2. From the **Source attribute** drop-down box, select `user.email`.
-    3. Click **Save**.
-6. Fill out the values required by Semgrep Cloud Platform's SAML2 form (**IdP SSO URL**, **IdP Issuer ID**, **X.509 Certificate**) by copying the values from Azure AD's **Set up Single Sign-On with SAML** page. Perform the following steps:
-    1. Copy the **Login URL** value from Azure AD to **IDP SSO URL** in Semgrep Cloud Platform. 
-    2. Copy **Azure AD Identifier** value to **IdP Issuer ID** in Semgrep Cloud Platform.
-    3. In **Set up Single Sign-On with SAML**, next to **Certificate (Base64)** click **Download**.
-    4. In Semgrep Cloud Platform, under **X.509 Certificate**, click **Browse** and then add the downloaded file. 
-    7. At the beginning of Semgrep Cloud Platform's SSO form, fill out **Email Domain** and **Display Name**.
-8. Click **Create/Update Auth Provider** at the beginning of of Semgrep Cloud Platform's SAML2 form.
-
-You have now set up SAML configuration between Azure AD and Semgrep Cloud Platform.
-
-#### Adding users to your new Enterprise App
+6. Click **Create**. This takes you to your new enterprise application's page.
+
+You have now created a custom enterprise app for Semgrep to integrate with Microsoft Entra ID. This enables you to set up SAML SSO.
+
+#### Set up SAML SSO for your new enterprise app
+
+1. From your new enterprise app's page, go to **Single-sign on** > **SAML**. 
+   ![Enterprise application's Single-sign on menu option](/img/entra-3.png#md-width)
+2. When prompted to **Select a single sign-on method**, select **SAML**. You are redirected to the **SAML-based Sign-on** page.
+   ![SAML-based Sign-on screen](/img/entra-4.png#md-width)
+3. In the **Basic SAML Configuration** section, click **Edit**. Provide the **Entity ID** and **Reply URL**. You can retrieve these values from Semgrep Cloud Platform by performing the following steps:
+    1. Log in to Semgrep Cloud Platform and navigate to [**Settings** > **Access** > **SSO**](https://semgrep.dev/orgs/-/settings/access/sso) page. 
+    2. Click **Add SAML2 SSO**.
+    3. Copy the **Audience URL** value from Semgrep Cloud Platform. Return to **Basic SAML Configuration**. Click **Add identifier** to paste this value as the **Identifier (Entity ID)**.
+    4. Copy the **SSO URL** value from Semgrep Cloud Platform. Return to **Basic SAML Configuration**. Click **Add reply URL** to paste this value as the **Reply URL (Assertion Consumer Service URL)**.
+4. Click **Save** and close out of **Basic SAML Configuration**.
+5. In the **Attributes and Claims** section, click **Edit**. You must add two claims. To add your first claim:
+    1. Click **Add new claim**.
+    2. Enter `name` in the **Name** field.
+    3. For the **Source attribute** drop-down box, select `user.givenname`.
```

Comment:
I know this is what we had before, but I wonder if we should actually be using `user.displayName`? The given name is just the first name which in many orgs is not enough to properly identify a user.